http-server/src/main/java/com/facebook/airlift/http/server/HttpServerBinder.java

```diff
@@ -70,5 +74,16 @@ public HttpResourceBinding withWelcomeFile(String welcomeFile)
             welcomeFiles.add(welcomeFile);
             return this;
         }
+
+        public HttpResourceBinding extraHeaders(String name, String value)
```

Comment:
```suggestion
        public HttpResourceBinding withExtraHeaders(String name, String value)
```

Reply:
updated it and changed it to the singular: `withExtraHeader`.
thanks!